lib/pharos/phases/configure_etcd.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Pharos
+  module Phases
+    class ConfigureEtcd < Base
+      CA_PATH = '/etc/pharos/pki'
+
+      register_component(
+        Pharos::Phases::Component.new(
+          name: 'etcd', version: Pharos::ETCD_VERSION, license: 'Apache License 2.0'
+        )
+      )
+
+      # @param host [Kupo::Configuration::Host]
+      # @param config [Kupo::Configuration]
+      def initialize(host, config)
+        @host = host
+        @config = config
+        @ssh = Pharos::SSH::Client.for_host(@host)
+      end
+
+      def call
+        sync_ca unless @host == @config.etcd_leader
+
+        logger.info(@host.address) { 'Configuring etcd certs ...' }
```

Comment:
The `Pharos::Phase` base already sets the address as logger progname (using it this way with the central logger would be better, having a separate logger for each phase makes it thread unsafe)